config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openshift-ai-toolkit/openshift-ai-toolkit-operator-controller
+  newName: quay.io/dbhagava/openshift-ai-toolkit-operator-controller
```

Comment:
this is not required. Changes to push to personal quay account is highly discouraged